test/async-hooks/test-graph.tls-write.js

```diff
@@ -20,8 +20,8 @@ hooks.enable();
 //
 const server = tls
   .createServer({
-    cert: fs.readFileSync(common.fixturesDir + '/test_cert.pem'),
-    key: fs.readFileSync(common.fixturesDir + '/test_key.pem')
+    cert: fs.readFileSync(`${common.fixturesDir}/test_cert.pem`),
```

Comment:
Sure, no problem on that.
So if I touch string concatenation that used as path next time, `path.join` is preferred rather than  template string syntax, right?